src/helpers/preprocess_text.py

```diff
@@ -116,6 +116,23 @@ def extract_subtext(text, key, index=0):
         return []
 
 
+def extract_html_tag(text: str, name: str) -> list:
```

Comment:
In the first commit you make a change to use this method, but I think I would swap the order of commits: so add this method with tests (nothing breaks) in first commit, then in a following commit make the fix to use this new code.

Reply:
Good tip, I'll rebase!

Reply:
Agree, changed the name of the function and also updated the commit message now. 👍